atlite/wind.py

```diff
@@ -90,3 +101,105 @@ def extrapolate_wind_speed(ds, to_height, from_height=None):
                           "units" : "m s**-1"})
 
     return wnd_spd.rename(to_name)
+
+def download_turbineconf(turbine=None, store_locally=True):
+    """Download a windturbine configuration from the OEDB database.
+    
+    Download the configuration of a windturbine model from the OEDB database
+    into the local 'windturbine_dir'.
+    The OEDB database can be viewed here:
+    https://openenergy-platform.org/dataedit/view/supply/turbine_library
+    (2019-07-22)
+    Only one turbine configuration is downloaded at a time, if the 
+    search parameters yield an ambigious result, not data is downloaded.
+    
+    Parameters
+    ----------
+    turbine : dict
+        Search parameters, either provide the turbine model id (takes priority)
+        or a manufacturer and turbine name, e.g. the following are identical:
+        {'id':10}, {'id':10, 'manufacturer':'Unknown', name:'E-53/800'},
+        {'manufacturer':'Enercon', name:'E-53/800'}
+    store_locally : bool
+        (Default: True) Whether the downloaded config should be stored locally
+        in config.windturbine_dir.
+        
+    Returns
+    -------
+    turbineconf : dict
+        The turbine configuration downloaded and stored is also returned as a dict.
+        Has the same format as returned by 'atlite.ressource.get_turbineconf(name)'.
+    """
+    
+    try: 
+        # Get the turbine list
+        OEDB_URL = 'https://openenergy-platform.org/api/v0/schema/supply/tables/turbine_library/rows'
+        result = requests.get(OEDB_URL)
+    except requests.exceptions.RequestException as e:
+        logger.info(f"Connection to OEDB failed with:\n\n{str(e)}")
+        return None
+
+    # Convert JSON to dataframe for easier filtering
+    # Only consider turbines with power curves available
+    df = pd.DataFrame.from_dict(result.json())
+    df = df[df.has_power_curve]
```

Comment:
Let's store this DataFrame somewhere in the `atlite` module to avoid hammering the db; ie. `_oedb_turbines = None`, on the module level.

Here
```python
global _oedb_turbines

if _oedb_turbines is None:
    [... download ...]

Reply:
Good idea!